torch/csrc/jit/passes/quantization.cpp

```diff
@@ -172,16 +173,32 @@ Node* createIntReprNode(Value* v, Graph* g) {
   return intrepr;
 }
 
+class ObserveHelper {
```

Comment:
Bikeshedding: maybe `InsertObserversHelper` to use the same pair of words everywhere?